eks-distro-base/check_update.sh

```diff
@@ -25,7 +25,7 @@ mkdir eks-distro-base/check-update
 cat << EOF >> eks-distro-base/check-update/Dockerfile
 FROM $BASE_IMAGE AS base_image
 
-RUN yum check-update --security
+RUN (yum check-update --security) && true
```

Comment:
This doesn't make much sense since true won't be run if yum fails. Does the conditional cause it to actually write ./return_value?

Reply:
Yes, I tested this construct with a simple script. The && true makes sure that the script doesn't fail, while retaining the return value and writing it to file.

```bash
set -e
yum check-update && true
echo "EXIT CODE IS "$?
echo "Hello world"
```

Output was
```bash
Loaded plugins: priorities
514 packages excluded due to repository priority protections
libgtop2.x86_64                                                                              2.38.0-3.amzn2                                                               amzn2-core
nice-dcv-cloud-dev-desktop.x86_64                                                            2020.2.72-1.el7                                                              dcv       
nice-dcv-server.x86_64                                                                       2020.2.9662-1.el7                                                            dcv       
nice-xdcv.x86_64                                                                             2020.2.359-1.el7                                                             dcv       
EXIT CODE IS 100
Hello world
```